api/handlers/annotations.go

```diff
@@ -34,23 +34,27 @@ LICENSE
 package handlers
 
 import (
-	"fmt"
 	"time"
 
-	"github.com/ausocean/openfish/api/utils"
+	"github.com/ausocean/openfish/api/api"
 
 	"github.com/gofiber/fiber/v2"
 )
 
+// TimeSpan is the json format for a pair of timestamps - start time and end time.
 type TimeSpan struct {
 	From time.Time `json:"from"`
 	To   time.Time `json:"to"`
 }
 
+// BoundingBox is the json format for a rectange enclosing something interesting in a video.
+// It is represeted using two x y coordinates, top left corner and bottom right corner of the rectange.
```

Comment:
Represented*

I'm beginning to think you should install a spell checker for your IDE.

Reply:
*rectangle